src/AppConstants.js

```diff
@@ -62,67 +62,69 @@ export const SEVERITY_MAP = {
     'medium-risk': 2,
     'low-risk': 1
 };
+// intlHelper seems to squash the use of HTML rich-text formatting
```

Comment:
naop theres a way to do this 🤤  checkout `rulesDetailsTotalriskBody || overviewActionCallNoSystemsBody || overviewDeployBody ||` in messages.js

🦖 🎉

Reply:
Oooo I'll check this out 🎉 :godmode:

Reply:
So looked at those messages... and _think_ the messages below are formatted the same way. It seems like the `intlProvider` behind `intlHelper` doesn't treat those HTML strings the same way as the `intlProvider` we are using in Advisor. (e.g. it will load as something like `<strong>low</strong>` rather than **low** 😬 )

Reply:
mmmmm so if intlHelper wont do the formatting, lets switch to the other one, gotta keep things translated

Reply:
@LI0131 gotta switch these over 😬 🤗

Reply:
Wait what did you previous comment mean? I think I misunderstood. Do we want to try to use the `intlProvider` from advisor? That's kind of what I'm doing now, by moving translation into the RuleDetails component. Butttt there could be a better way of doing it

Reply:
Had to import `React` for the HTML to work 💀 😩